$DragonFly: src/gnu/usr.bin/groff/tmac/patches/tty-char.tmac.no_obj.patch,v 1.1 2005/09/01 20:47:14 asmodai Exp $

--- tty-char.tmac	2005-08-31 14:28:07.000000000 +0200
+++ tty-char.tmac	2005-08-31 15:49:48.000000000 +0200
@@ -233,5 +233,7 @@
 .
 .ie '\*(.T'cp1047' \
 .	do mso cp1047.tmac
+.el .ie '\*(.T'koi8-r' \
+.      do mso koi8-r.tmac
 .el \
 .	do mso latin1.tmac
